tests/test_operations.py

```diff
@@ -51,28 +51,20 @@ def test_to_rna_2():  # Reference: https://biomodel.uah.es/en/lab/cybertory/anal
     assert oprimer_rna == "AUCGAUCGAUCG"
 
 
-def test_to_protein():  # Reference: https://biomodel.uah.es/en/lab/cybertory/analysis/trans.htm
+def test_to_protein_1(): # https://en.vectorbuilder.com/tool/dna-translation.html
     oprimer = Primer("ATCGATCG")
-    oprimer_protein = oprimer.to_protein()
-    assert oprimer_protein == "ID"
+    assert oprimer.to_protein() == "ID" and oprimer.to_protein(frame=2) == "SI" and oprimer.to_protein(frame=3) == "RS"
+    assert oprimer.to_protein(one_letter=False) == "lle-Asp" and \
+        oprimer.to_protein(frame=2, one_letter=False) == "Ser-lle" and \
+        oprimer.to_protein(frame=3, one_letter=False) == "Arg-Ser"
```

Comment:
Good idea to merge them under the same unit test. Please make these assertions in different lines for more readability and future error detection. Do this both for `test_to_protein_1` and `test_to_protein_2`.